src/Ng/Catalog2Dnx.cs

```diff
@@ -49,7 +49,23 @@ public Catalog2Dnx(ILoggerFactory loggerFactory)
 
         private static void PrintUsage()
         {
-            Console.WriteLine("Usage: ng catalog2dnx -source <catalog> -contentBaseAddress <content-address> -storageBaseAddress <storage-base-address> -storageType file|azure [-storagePath <path>]|[-storageAccountName <azure-acc> -storageKeyValue <azure-key> -storageContainer <azure-container> -storagePath <path>] [-verbose true|false] [-interval <seconds>]");
+            Console.WriteLine("Usage: ng catalog2dnx "
+                + "-"  + CommandHelpers.Source             + " <catalog> "
+                + "-"  + CommandHelpers.ContentBaseAddress + " <content-address> "
```

Comment:
nit: use C# 6 formatting. That way the string creation will look even cleaner:
$"-{CommandHelpers.Source} <...>"